apollo-router-core/src/json/mod.rs

```diff
@@ -0,0 +1,30 @@
+//! custom JSON `Value` implementation, adapted to use `bytes::Bytes` internally.
```

Comment:
:yarn: probably better to group with the module json_ext. It's all related to extending the handling of json with serde.

Reply:
yes, it makes sense to have them in the same module. I was keeping them separated to make the experimentation easier (that PR is still in draft :sweat_smile:)

Reply:
oh sorry xD